run.py

```diff
@@ -239,6 +242,20 @@ def keyboard_click(*KEYBOARD_NAMES):
 def grab_screen(n):
     return callback.get_screen_grab()
 
+@app.callback(
+    Output('calendar-screen','children'),
+    [Input('next-change_mth', 'n_clicks')])
+def next_month(n_clicks):
+    if n_clicks > 0:
+        change_next_month() 
+
+@app.callback(
+    Output('live-calendar','children'),
+    [Input('calendar-interval-component', 'n_intervals')])
+def grab_calendar(n):
+   change_calendar_content() 
+   create_all_calendar()
```

Comment:
There's no `return`,  so I assume that this function return **None** to live-calendar